cmd/kops/create.go

```diff
@@ -125,11 +129,19 @@ func RunCreate(f *util.Factory, out io.Writer, c *CreateOptions) error {
 	var sb bytes.Buffer
 	fmt.Fprintf(&sb, "\n")
 	for _, f := range c.Filenames {
-		contents, err := vfs.Context.ReadFile(f)
-		if err != nil {
-			return fmt.Errorf("error reading file %q: %v", f, err)
+		var contents []byte
+		if f == "-" {
+			file := os.Stdin
+			defer file.Close()
```

Comment:
Nit: I don't think we need to close stdin. 

In general, a `defer` in a loop is a code-smell because it is deferred until the function exit, not the loop iteration exit.  But in this case, I think it's fine!